clients/rospy/src/rospy/core.py

```diff
@@ -131,18 +131,10 @@ def parse_rosrpc_uri(uri):
 # other sorts of information that scare users but are essential for
 # debugging
 
-def rospydebug(msg, *args):
-    """Internal rospy client library debug logging"""

```

Comment:
The information that these functions are "internal" is being lost in the patch. Based on the fact that the functions are intended to be internal I am wondering why you want to allow `kwargs` here? Anyway if it makes still sense to allow them here please update the patch to add them to the existing function definitions and pass them along.